generators/java/okhttp-gson/templates/README.mustache

```diff
@@ -155,29 +155,8 @@ try {
 }
 ```
 
-
 ### Recommendation
 
 It's recommended to create an instance of `ApiClient` per thread in a multithreaded environment to avoid any potential issues.
 
-## Contributing
-
-This library is automatically generated using [OpenAPI Generator](https://openapi-generator.tech) - version: {{generatorVersion}}; therefore all the contributions, except tests files, should target [Onfido OpenAPI specification repository](https://github.com/onfido/onfido-openapi-spec/tree/master) instead of this repository.
-
-For contributions to the tests instead, please follow the steps below:
-
-1. [Fork](<https://github.com/{{ gitUserId }}/{{ gitRepoId }}/fork>) repository
```

Comment:
The `<` and `>` here were wrong (even if fixed by CI via `npx prettier --write *.md`)